api/test/pactUtil.ts

```diff
@@ -6,41 +6,38 @@ import * as path from 'path';
 let mockServerPort: number;
 let provider: Pact;
 
-export async function providerSetUp(providerName:string) {
+export async function providerSetUp(providerName: string) {
   mockServerPort = await getPort()
 
   provider = new Pact({
-    port: mockServerPort,
-    log: path.resolve(process.cwd(), "api/test/pact/logs", "mockserver-integration.log"),
-    dir: path.resolve(process.cwd(), "api/test/pact/pacts"),
-    spec: 2,
     consumer: "xui_approveorg",
-    provider: providerName,
+    dir: path.resolve(process.cwd(), "api/test/pact/pacts"),
+    log: path.resolve(process.cwd(), "api/test/pact/logs", "mockserver-integration.log"),
     pactfileWriteMode: "merge",
+    port: mockServerPort,
+    provider: providerName,
+    spec: 2,
   })
   before(() => provider.setup())
 
 //  return  provider;
 }
 
-
 export async function providerFinalize() {
   after(() => provider.finalize())
   // verify with Pact, and reset expectations
   afterEach(() => provider.verify())
 //  return  provider;
 }
 
-export async  function addInteraction(interaction:InteractionObject){
+export async  function addInteraction(interaction: InteractionObject) {
 
   // @ts-ignore
   provider.addInteraction(interaction).then(() => {
-
+    // Do nothing, apparently.
```

Comment:
does this comment add value? Pretty clear that nothing happens..

Reply:
If it's not here, there's a lint error. 🤷‍♂️

Reply:
You mean ditch the whole `.then()`?

Reply:
I've tried not to touch the PACT tests because they're already broken. The only thing I've done at this point is fix the lint errors.